datastore/sync_entity.go

```diff
@@ -85,17 +95,37 @@ func (a TagItemByClientIDID) Less(i, j int) bool {
 	return a[i].ClientID+a[i].ID < a[j].ClientID+a[j].ID
 }
 
+// getTagPrefix is a helper method to give the proper prefix for unique tag
+func getTagPrefix(isServer bool) string {
+	if isServer {
+		return serverTagItemPrefix
+	} else {
+		return clientTagItemPrefix
+	}
+}
+
 // NewServerClientUniqueTagItem creates a tag item which is used to ensure the
 // uniqueness of server-defined or client-defined unique tags for a client.
 func NewServerClientUniqueTagItem(clientID string, tag string, isServer bool) *ServerClientUniqueTagItem {
```

Comment:
Shouldn't the unit test needs to be updated?
Also I noticed there seem to be no CI tests being run for the PR, not sure what happens here?